tests/ansible_module_playook_test.go

```diff
@@ -0,0 +1,47 @@
+package tests_test
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+)
+
+const (
+	ansibleModuleURL          = "https://github.com/kubevirt/ansible-kubevirt-modules/"
```

Comment:
Maybe you should consider to download only latest tag, in order to have the test/job stable.

Reply:
`go-git` gets the master by-default, and master is latest for us